test/fixtures/replace-multiple-object-assign-calls/expected.js

```diff
@@ -1,10 +1,17 @@
 "use strict";
 
-var _simpleAssign = require("simple-assign");
-
-var _simpleAssign2 = _interopRequireDefault(_simpleAssign);
+var _simpleAssign = _interopRequireDefault(require("simple-assign"));
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-(0, _simpleAssign2.default)({ a: 1 }, { b: 2 });
-(0, _simpleAssign2.default)({ c: 3 }, { d: 4 });
+_objectAssign({
```

Comment:
@jaydenseric Should this be `_simpleAssign` if this is working correctly?

Reply:
I'm super confused. How come the tests used to pass with `_simpleAssign` expected, when the [UID was hardcoded to `objectAssign` all along](https://github.com/newoga/babel-plugin-transform-replace-object-assign/blob/v0.2.0/src/index.js#L28)?

Reply:
Yeah, me too. It's be a while since I have looked at this (and babel plugin development in general). But it's possibly that in babel v6, babel always generated the UID for an import based on the package [here](https://github.com/newoga/babel-plugin-transform-replace-object-assign/blob/v0.2.0/src/index.js#L17). The `generateUidIdentifier('objectAssign')` may have been unnecessary/unused. Can't confirm that at the moment.